### PR TITLE
[NUI] Add FindDescendantByID(), Raise(), Lower(), and etc by request of user

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Add FindDescendantByID(), Raise(), Lower(), and etc by request of user
- This is ACR patch
- https://code.sec.samsung.net/jira/browse/GRE-2303

### API Changes ###
https://code.sec.samsung.net/jira/browse/TCSACR-438

### CODE REVIEW

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs [hunk 1]
```diff
@@ -499,8 +497,7 @@ public static Size2D GetClosestImageSize(string filename)
         /// </summary>
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/Images/ImageLoading.cs:497]: "consider rotation" -> "taking the image rotation into account" OR ", rotated according to image metadata".

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs [hunk 2]
```diff
@@ -23,9 +23,7 @@ namespace Tizen.NUI
     /// <summary>
     /// Loading an image.
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/Images/ImageLoading.cs:24]: maybe better sth like "A set of tools providing image loading", "A class containing methods to load an image from file ..."

File: src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs [hunk 3]
```diff
@@ -662,17 +662,144 @@ public void ObjectDump()
...
         /// <summary>
         /// Search through this View's hierarchy for a View with the given unique ID.
-        /// The View itself is also considered in the search.
         /// </summary>
-        /// <param name="id">The ID of the View to find</param>
-        /// <returns>A View if found or a null if not</returns>
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public View FindChildByID(uint id)
+        /// <param name="id">The ID of the View to find.</param>
+        /// <remarks>Hidden-API</remarks>
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs:667]: Hidden? Everybody will see it. It was hidden by EditorBrowsableState.Never

File: src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs [hunk 4]
```diff
@@ -662,17 +662,144 @@ public void ObjectDump()
...
+                }
+            }
+
+        }
+
+        /// <summary>
+        /// Lowers the view to below the target view.
+        /// </summary>
+        /// <remarks>The sibling order of views within the parent will be updated automatically.
+        /// Lowering this view below views with the same sibling order as each other will lower this view above them.
```
Comment (wiertel) [inline src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs:770]: above -> below,
I don't get the "as each other", multiple views can have same sibling order? maybe say "Lowering this view below a view which shares sibling order with others / has same sibling order as other views, will ..."

File: src/Tizen.NUI/src/public/Images/ImageLoading.cs (hunk 1 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Images/ImageLoading.cs:497]: thank you for review.
it's done

**Review by huiyueun — APPROVED** [review]

**Pull request merged**